src/components/LottieControl.jsx

```diff
@@ -1,12 +1,28 @@
 import React, { Component } from 'react'
 import Lottie from 'react-lottie';
-import * as animationData from '../assets/animations/simpleBlob.json'
+import * as lucyIdle from '../assets/animations/lucyIdle.json';
+import * as lucyColors from '../assets/animations/lucyColors.json';
+import * as lucyBowSpin from '../assets/animations/lucyBowSpin.json';
 
 export default class LottieControl extends Component {
 
   constructor(props) {
     super(props);
-    this.state = {isStopped: false, isPaused: false};
+    this.state = {
+      isStopped: false, 
+      isPaused: false,
+      animationsArray: [lucyIdle, lucyColors, lucyBowSpin],
+      currentAnimation: null,
+      index: 0,
```

Comment:
Maybe we can rename this to be a bit more descriptive? Something like `currentAnimationIndex`.